zwave_js/config.json

```diff
@@ -25,6 +26,7 @@
     "device": "device(subsystem=tty)",
     "network_key": "match(|[0-9a-fA-F]{32,32})",
     "log_level": "list(silly|debug|verbose|http|info|warn|error)?",
+    "custom_config_dir": "str?",
```

Comment:
Same discussion as in:

<https://github.com/home-assistant/addons/issues/1410>


This should not be added until: <https://github.com/home-assistant/supervisor/issues/1842>
is resolved.

Reply:
boo ok 🙁